nodes/nodes/eco-node-health-check-operator.adoc

```diff
@@ -21,3 +21,8 @@ include::modules/eco-node-health-check-operator-about.adoc[leveloffset=+1]
 include::modules/eco-node-health-check-operator-installation-web-console.adoc[leveloffset=+1]
 
 include::modules/eco-node-health-check-operator-installation-cli.adoc[leveloffset=+1]
```

Comment:
link doesn't work

Reply:
![image](https://user-images.githubusercontent.com/58551443/154069999-e29075a3-d770-480f-aa4c-d5afe3d1a32a.png)

https://github.com/openshift/openshift-docs/blob/main/nodes/nodes/modules/eco-node-health-check-operator-installation-cli.adoc

Reply:
The link works in the browser. Check this: https://deploy-preview-41724--osdocs.netlify.app/openshift-enterprise/latest/nodes/nodes/eco-node-health-check-operator.html#installing-node-health-check-operator-using-cli_node-health-check-operator

Reply:
this link works